src/microservices/Microservices.CohortPackager/Options/CohortPackagerCliOptions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using CommandLine;
+using CommandLine.Text;
+using JetBrains.Annotations;
+using Smi.Common.Options;
+using System;
+using System.Collections.Generic;
+
+
+namespace Microservices.CohortPackager.Options
+{
+    public class CohortPackagerCliOptions : CliOptions
+    {
+        [Option(
+            'r',
+            "recreate-report",
+            Required = false,
+            HelpText = "[Optional] Recreate the report for the specified extraction ID"
+        )]
+        public Guid ExtractionId { get; set; }
+
+        [Usage]
+        [UsedImplicitly]
+        public static IEnumerable<Example> Examples
+        {
+            get
+            {
+                yield return new Example("Normal Scenario", new CohortPackagerCliOptions { ExtractionId = Guid.NewGuid() });
```

Comment:
Is supplying the guid the "Normal Scenario"? Maybe provide 2 examples 1 in microservice mode 1 in recreate mode.

Reply:
Have clarified this now.